DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy.recipe.json

```diff
@@ -2505,25 +2506,10 @@
                                 "ContentIndexSettings": {},
                                 "TaxonomyFieldSettings": {
                                     "Required": true,
-                                    "TaxonomyContentItemId": "4eembshqzx66drajtdten34tc8",
+                                    "TaxonomyContentItemId": "4w9kg8jhved372sg703ggkq5wx",
```

Comment:
You'll need to revert the id to the old value (so that it matches the id the taxonomy gets created with through the taxonomy recipe).